canal/sync.go

```diff
@@ -187,6 +187,7 @@ func (c *Canal) handleRowsEvent(e *replication.BinlogEvent) error {
 		return errors.Errorf("%s not supported now", e.Header.EventType)
 	}
 	events := newRowsEvent(t, action, ev.Rows)
+	events.NextPos = nextPos
```

Comment:
why not passing nextPos as the param of newRowsEvent?

Reply:
ok, I will follow your guide